django/contrib/admin/options.py

```diff
@@ -1022,7 +1023,7 @@ def construct_search(field_name):
         if search_fields and search_term:
             orm_lookups = [construct_search(str(search_field))
                            for search_field in search_fields]
-            for bit in search_term.split():
+            for bit in shlex.split(search_term):
```

Comment:
I don't think that we want to split a string using shell-like syntax. We should rather use `smart_split()` and `unescape_string_literal()` like proposed in the ticket, e.g.
```python
for bit in smart_split(search_term):
    bit = unescape_string_literal(bit) if bit.startswith(('"', "'")) else bit
    ...
```

```